server/routerlicious/packages/services-core/src/database.ts

```diff
@@ -87,7 +87,7 @@ export interface ICollection<T> {
      * @param query - data we want to find
      * @param value - data to update to the database
      */
-    findAndUpdate(query: any, value: T): Promise<{
+    findAndUpdate(query: any, value: any): Promise<{
```

Comment:
Why change this to `any`?

Reply:
The MongoDB interface has been changed. It would give us this error if we don't change it to `any`. 
![MicrosoftTeams-image](https://user-images.githubusercontent.com/66701969/223555660-927673c3-3d27-46db-ad62-a17b5cc221e4.png)